cypress-tests/cypress/e2e/checkWorkflows.cy.ts

```diff
@@ -0,0 +1,124 @@
+import { TIMEOUTS } from "../../config";
+import { ProjectIdentifier } from "../support/commands/projects";
+import { validateLogin } from "../support/commands/general";
+
+const project: ProjectIdentifier = {
+  namespace: "renku-ui-tests",
+  name: "composite-workflows",
+};
+
+describe("Workflows pages", () => {
+  before(() => {
+    // Use a session to preserve login data
+    cy.session(
+      "login-updateProjects",
+      () => {
+        cy.robustLogin();
+      },
+      validateLogin
+    );
+  });
+
+  beforeEach(() => {
+    // Restore the session
+    cy.session(
+      "login-updateProjects",
+      () => {
+        cy.robustLogin();
+      },
+      validateLogin
+    );
+  });
+
+  it.only("Check workflows", () => {
+    cy.visitAndLoadProject(project);
+
+    // Go the the workflows page and check the details of a workflow
+    cy.dataCy("project-navbar")
+      .contains("a.nav-link", "Workflows")
+      .should("be.visible")
+      .click();
+
+    cy.get("[data-cy=workflows-page]", { timeout: TIMEOUTS.long }).should(
+      "be.visible"
+    );
+    cy.dataCy("workflows-browser")
+      .should("be.visible")
+      .children()
+      .should("have.length", 9);
+    cy.dataCy("workflows-browser")
+      .children()
+      .contains("useless-workflow-with-kw")
+      .should("be.visible")
+      .click();
```

Comment:
I think this is a bit easier to follow without reselecting the `workflows-browser`

```suggestion
    cy.dataCy("workflows-browser")
      .should("be.visible")
      .children()
      .should("have.length", 9)
      .contains("useless-workflow-with-kw")
      .should("be.visible")
      .click();
```